coalib/results/Diff.py

```diff
@@ -285,10 +317,11 @@ def unified_diff(self):
         Note that the unified diff is not deterministic and thus not suitable
         for equality comparison.
         """
-        return ''.join(difflib.unified_diff(
-            self.original,
-            self.modified,
-            tofile=self.rename if isinstance(self.rename, str) else ''))
+        tofile = (self.rename if isinstance(self.rename, str) else
+                  self.create if isinstance(self.create, str) else '')
+
+        return ''.join(difflib.unified_diff(self.original, self.modified,
```

Comment:
There is no need to modify *all* of the original lines.  Only one line needs to be modified.  lines 288-290 can be unmodified, and then line 291 becomes `tofile=tofile`